tests/Nox.ClientApi.Tests/Tests/Controllers/CountriesControllerTests.cs

```diff
@@ -240,6 +241,96 @@ public async Task Post_WithManyOwnedEntity_ReturnsAutoNumberId()
         getCountryResponse!.CountryBarCode.Should().NotBeNull();
         getCountryResponse!.CountryBarCode!.BarCodeName.Should().Be(expectedBarCodeName);
     }
+    
+    [Fact(Skip = "Skipped due to postgresql cascade delete not working as expected, will be fixed in the future.")]
```

Comment:
I would prefer to use the `#if RELEASE`. Because on the CI we run the tests in release for SQL Server and in this context they should pass. See for example test `Put_WithCountryLocalNames_FromListToEmpty_Success`

Reply:
Desired changes have been applied